Utils/data_analys.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import pandas as pd
+import spacy
+import re
+from collections import Counter
+from nltk import ngrams
+from nltk.tokenize import TweetTokenizer
+
+
+DATA_DIR = '../../Data/Russian Twitter Corpus/'
+
+
+def read_csv_clear(name):
+    '''
+    Читает данные из файла svc, и преобразует их для дальнейшей обработки
+    :param name: имя файла
+    :return: dataframe с подготовленными данными
+    '''
+    x = pd.read_csv(f"{DATA_DIR}" + name, sep=';', header=None, engine='c')
+    tknzr = TweetTokenizer(strip_handles=True, reduce_len=True)
+    for i in range(12):
+        if i != 3:
+            del x[i]
+    for i,_ in enumerate(x[3]):
+        x.at[i, 3] = ' '.join(tknzr.tokenize(x.at[i, 3]))
+
+    x['size'] = pd.Series(len(x[3]), index=x.index)
+    for i, _ in enumerate(x['size']):
+        x.at[i, 'size'] = len(x.at[i, 3])
+
+    reg = re.compile('[^a-zA-Z ][^а-яА-Я ]]')
```

Comment:
Let's move this regular expression up to the all methods as an magic constant :)